contents/docs/libraries/js/index.mdx

```diff
@@ -303,6 +303,66 @@ posthog.reloadFeatureFlags()
 Note that `reloadFeatureFlags()` does not send `$feature_flag_called` events. These events are only sent when you query for a feature flag
 using the `getFeatureFlag()` or `isFeatureEnabled()` functions.
 
+
+#### Overriding server properties
+
+Sometimes, you might want to evaluate feature flags using properties that haven't been ingested yet, or were set incorrectly earlier. You can do so by setting properties the flag depends on with these calls:
+
+```js
+posthog.personPropertiesForFlags({'property1': 'value', property2: 'value2'})
+```
+
+Note that these are set for the entire session, and successive calls are additive: all properties you set are combined together and sent for flag evaluation.
+
+Whenever you set these properties, we also trigger a reload of feature flags to ensure we have the latest values. You can disable this by passing in the optional parameter for reloading:
+
+```js
+posthog.personPropertiesForFlags({'property1': 'value', property2: 'value2'}, false)
+```
+
+At any point, you can reset these properties by calling `resetPersonPropertiesForFlags`:
+
+```js
+posthog.resetPersonPropertiesForFlags()
+```
+
+The same holds for [group](/manual/group-analytics) properties:
+
+
+```js
+// set properties for a group
+posthog.groupPropertiesForFlags({'organization': {'property1': 'value', property2: 'value2'}})
+
+// reset properties for a given group:
+posthog.resetGroupPropertiesForFlags('organization')
+
+// reset properties for all groups:
+posthog.resetGroupPropertiesForFlags()
+```
```

Comment:
```suggestion
```


> **Note:** You don't need to add the group names here, since these properties are automatically attached to the current group (set via `posthog.group()`). When you change the group, these properties are reset.
```